src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/TakeCard.kt

```diff
@@ -0,0 +1,111 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIcon
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIconView
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.event.ActionEvent
+import javafx.geometry.Pos
+import javafx.scene.layout.AnchorPane
+import javafx.scene.layout.Pane
+import javafx.scene.layout.Priority
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.common.device.IAudioPlayer
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
+import tornadofx.*
+
+class TakeCard(take: Take, player: IAudioPlayer) : AnchorPane() {
+    private val badge = stackpane {
+        style {
+            backgroundColor += c(Colors["primary"])
+            backgroundRadius += box(0.px, 10.px, 0.px, 10.px)
+            padding = box(8.px)
+        }
+        val icon = MaterialDesignIconView(MaterialDesignIcon.CREATION)
+        icon.size = "18px"
+        icon.style(true) {
+            fill = Color.WHITE
+        }
+        add(icon)
+        isVisible = !take.played
+    }
+
+
+    init {
+        setRightAnchor(badge, 0.0)
+        setTopAnchor(badge, 0.0)
+        style {
+            minWidth = 250.px
+            maxWidth = minWidth
+            minHeight = 100.px
+            maxHeight = minHeight
+            backgroundColor += Color.WHITE
+            backgroundRadius += box(10.px)
+        }
+        val content = vbox {
+            style {
+                padding = box(10.px)
+            }
+            hbox {
+                vgrow = Priority.ALWAYS
+                style {
+                    spacing = 10.px
```

Comment:
spacing for the hbox can also be specified like so 

`hbox(10.0) {} `

Reply:
Good to know. Thanks!